doc/js_tutorials/js_imgproc/js_pyramids/js_pyramids.markdown

```diff
@@ -10,15 +10,15 @@ Goal
 Theory
 ------
 
-Normally, we used to work with an image of constant size. But in some occassions, we need to work
-with images of different resolution of the same image. For example, while searching for something in
-an image, like face, we are not sure at what size the object will be present in the image. In that
-case, we will need to create a set of images with different resolution and search for object in all
-the images. These set of images with different resolution are called Image Pyramids (because when
-they are kept in a stack with biggest image at bottom and smallest image at top look like a
-pyramid).
+Normally, we used to work with an image of constant size. But on some occasions, we need to work
+with (the same) images in different resolution. For example, while searching for something in
+an image, like face, we are not sure at what size the object will be present in said image. In that
+case, we will need to create a set of the same image with different resolutions and search for object
+in all of them. These set of images with different resolutions are called **Image Pyramids** (because
+when they are kept in a stack with the largest resolution image at the bottom and the lowest resolution
+image at top, it looks like a pyramid).
 
-There are two kinds of Image Pyramids. 1) Gaussian Pyramid and 2) Laplacian Pyramids
+There are two kinds of Image Pyramids. 1) **Gaussian Pyramid** and 2) **Laplacian Pyramids**
```

Comment:
I also got a little creative, perhaps too much? Let me know if this isn't appropriate and I'll revert it.